README.md

```diff
@@ -7,7 +7,10 @@ LOCAL_POSTGRES_PASSWORD: 'YYY'
 
 With your local postgres username/password properly filling in the blanks.
 
+Note: methods like admin? and student? are no longer supported. Use admin_role instead. See models/role.rb
```

Comment:
Have you tried using "alias" to just make `admin?` a direct copy of `admin_role`? It would minimize the effort of needing to revise old code.

Reply:
if the old version is still out there we could do this, but I don't have it on my machine anymore